docs/custom_promise_types/git_using_lib.py

```diff
@@ -14,9 +14,6 @@ def validate_promise(self, promiser, attributes):
                 raise ValidationError(f"'repo' must be string for git promise types")
 
     def evaluate_promise(self, promiser, attributes):
-        if not promiser.startswith("/"):
-            raise ValidationError("File path must be absolute")
-
```

Comment:
Looks like this should be part of the previous commit.

Reply:
It could be in both, I did this intentionally. Since this commit introduces the part of the spec that says you should not validate during evaluation.